ckanext/sse/blueprints/data_reuse.py

```
@@ -0,0 +1,303 @@
+            if (
+                submission.get("package_id")
+                and submission["package_id"] not in package_names
+            ):
+                try:
+                    pkg = tk.get_action("package_show")(
+                        context, {"id": submission["package_id"]}
+                    )
+                    package_names[submission["package_id"]] = pkg.get(
+                        "title", pkg.get("name", "")
+                    )
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)